src/components/GameCards/GameCardCover.vue

```diff
@@ -6,7 +6,7 @@
             @click="openDetails"
         >
 
-        <i class="fas fa-grip-vertical game-drag-handle" />
+        <i v-bind:class="{ 'fas fa-grip-vertical game-drag-handle' : list.sortOrder == 'sortByCustom' || list.sortOrder == null }" />
```

Comment:
I think you should use `v-if` here so the item doesn't render at all if it's not manually sortable
```suggestion
<i
    v-if="list.sortOrder == 'sortByCustom' || list.sortOrder == null"
    class="fas fa-grip-vertical game-drag-handle"
/>
```

also if template logic gets too long consider moving it to a computed prop like `isDraggable` or similar

```suggestion
<i
    v-if="isDraggable"
    class="fas fa-grip-vertical game-drag-handle"
/>
```